app/controllers/events_controller.rb

```diff
@@ -30,6 +30,7 @@ def edit
 
   def create
     @event = @room.events.new(event_params.merge(user_id: current_user.id))
+    EventMailer.new_event(@event).deliver_now
```

Comment:
My only issue with this is that it looks like it will always send the Event Mailer new event mail even if the event cannot be created and fails validations.